### PR TITLE
Add FAQ and Privacy Policy placeholders

### PR DESCRIPTION
**Pull request by donpui**
Adding placeholders for FAQ and Privacy Policy
Additionally small correction for Readme

It would be good to merge placeholder in order to have links, which we can use in other materials preparations

---

## Code Review Checklist

- [ ] Description accurately reflects what changes are being made.
- [ ] Description explains why the changes are being made (or references an issue containing one).
- [ ] The PR appropriately sized.
- [ ] New code has enough tests. 
- [ ] New code has enough documentation to answer "how do I use it?" and "what does it do?".
- [ ] Existing documentation is up-to-date, if impacted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Cross-platform Magic Wormhole graphical client
+# Destiny - Cross-platform graphical client based on Magic Wormhole
```
Comment (meejah) [inline README.md:1]: I think the previous sentence is better; this _is_ "a magic-wormhole client"

**Review by meejah — CHANGES_REQUESTED** [review]

**Review by meejah — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -52,7 +53,7 @@ Builds for local instances:
 flutter build linux lib/main_local.dart
 ```
 
-## Signing Android AppBundle for Google Play upload
+### Signing Android AppBundle for Google Play upload
```
Comment (ewanas) [inline README.md:56]: Can this section be moved to `doc` directory instead?

Comment (donpui) [reply README.md:56]: Yes, I will do

Comment (donpui) [reply README.md]: I made changes, however README.md I plan to change more with another PR

**Review by ewanas — APPROVED** [review]

**Pull request merged**